### PR TITLE
Create stale issues flow

### PR DESCRIPTION
**Pull request by shiranr**
## What are you trying to address

Our issues backlog is field with stale issues.
Give 90 days for an issue to be solved, when not solved, put a notification and when that doesn't work close it after 14 days.
#839 

## Description of new changes

Using github workflow to solve the issue

## For all pull requests

- [ ] Link to the issue you are solving (so it gets closed)
- [ ] Label the pull request with the appropriate area(s)
- [ ] Assign potential reviewers (you may also want to contact them on Teams to ensure timely reviews)
- [ ] Assign the project - Engineering Playbook Backlog
- [ ] Assign the pull request to yourself

### CODE REVIEW

**Review by TessFerrandez — CHANGES_REQUESTED** [review]
Setting it to request changes - I think we need to give the champs leads a heads up first before we comitt this.

File: .github/workflows/stale_issues_flow [hunk 1]
```diff
@@ -0,0 +1,23 @@
+name: Close inactive issues
+# At 00:00 on every day-of-week from Monday through Friday.
+on:
+  schedule:
+    - cron: "0 0 * * 1-5"
+
+jobs:
+  close-issues:
+    runs-on: ubuntu-latest
```
[the remark this answers is not in the document]
Comment (shiranr) [reply .github/workflows/stale_issues_flow:9]: So for now how about just tagging the issues as stale?

**Comment by TessFerrandez** [comment]
Closes #839 <- just adding this so the issue will be closed once the PR is merged

**Review by tompaana — APPROVED** [review]
LGTM!

**Pull request merged**